README.md

```diff
@@ -85,6 +86,8 @@ Use *Quick Install* option if you want to quickly install EAF source code and tr
 3. From here on, you can either Add the full path to the EAF installation directory to your Emacs ```load-path```, then add the following to `init.el`:
 
 ```Elisp
+;; For debugging uncomment Following
+;; (message "Loading [EAF] ...")
```

Comment:
This is very unnecessary

Reply:
got it, please feel free to remove it. Or let me know if I should revert the commits and make new one with updated info.

Reply:
You can simply add a new commit to your forked branch, it'll update it here automatically.

Reply:
got it, I shall do the same with both then.

Reply:
I shall close this one, and then open the one with required commits.